synapseutils/walk_functions.py

```diff
@@ -67,14 +67,15 @@ def walk(
     # Ensure that "folder" is included so the hierarchy can be traversed
     if "folder" not in includeTypes:
         includeTypes.append("folder")
-    return _helpWalk(syn, synId, includeTypes)
+    return _help_walk(syn=syn, syn_id=synId, include_types=includeTypes)
 
 
 # Helper function to hide the newpath parameter
-def _helpWalk(
+def _help_walk(
```

Comment:
Nit: Not sure what I was thinking when I was writing this, but the `_` is almost redundant with `help`.  I can imagine this could be `_walk`, but... this is good.  no action required.  Just a thought in the future for naming functions.